gather_vars

```diff
@@ -0,0 +1,11 @@
+extra_args=''
+
+if [ "$1" == "--rcm" ]; then
+  shift
+  extra_args+='-e @../../rcm_vars.yml'
```

Comment:
Yes, likely it will be a problem. I'll look at an alternative way of handling it.

Thanks for catching that.